### ISSUE TITLE
Aliasing of curves

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/calculus-grapher#242]
In https://github.com/phetsims/qa/issues/898#issuecomment-1437631257, @liammulh reported aliasing on iPad.  I'm not sure what we can do about this, since any diagonal line will always be somewhat more aliased.   And I'm not clear about whether it's specific to iPad. Assigning to @veillette and myself for discussion.

---
These notes apply to:

```
Tester = LM
Device =
    (Seemingly has no name. Also has no cart sticker. Black with black case. Lion logo on the case.)
    iPad Model MR7F2LL/A
    iPadOS 16.3
    Safari 16.3 Mobile
Time = 1.5 hour
```
...
The circled line in the below screenshot is a bit "fuzzy". This happens to the curves/lines in this sim a lot on the iPad I was using to test.

...
<img width="600" src="https://user-images.githubusercontent.com/60749003/220206173-d06ba090-fd80-4dc0-882a-f8240162d6b2.png" alt="weird"/>

**Comment by pixelzoom** [comment]
@liammulh Could you please comment on whether you see similar antialiasing on other platforms?  Or was this specific to iPad?

**Comment by liammulh** [comment]
I only saw it on the iPad.

**Comment by veillette** [comment]
`CurveManipulationDisplayNode` controls the display of the curve. 
```
    const curve = new TransformedCurve( {
      numberOfPoints: 300,
      xRange: CalculusGrapherConstants.CURVE_X_RANGE,
      tandem: Tandem.OPT_OUT
    } );
```
I'll note that the number of points for this display is 300, which could be increased, without impacting performance much. However, since the curve shown above is linear, it may not address the problem, although it may be worth trying.

Also increasing the linewidth may reduce the antialiasing.

**Comment by pixelzoom** [comment]
@veillette do you have an iPad to test the above hypotheses?  If not, I can have a go at it.

**Comment by veillette** [comment]
@pixelzoom: No I dont. Yes, give it a go.

**Comment by pixelzoom** [comment]
Zooming in on the screenshot that @liammulh provided, it's easier to see the aliasing.  I reproduced this on my iPad6.

<img width="437" alt="screenshot_2339" src="https://user-images.githubusercontent.com/3046552/222837021-f8b373c3-0e9d-4566-b529-691341d2db00.png">

**Comment by pixelzoom** [comment]
What's interesting is if I pinch-to-zoom on the iPad, the curve looks nice and crisp:

![IMG_A15D84B7B210-1](https://user-images.githubusercontent.com/3046552/222837629-5497e0f0-a2ab-4c83-85a4-3bef78c4ff9a.jpeg)

**Comment by pixelzoom** [comment]
Adding more points does not improve things.  With the default 300 points, I took a screenshot of the sim, then zoomed in on that screenshot in macOS Preview:

<img width="887" alt="screenshot_2342" src="https://user-images.githubusercontent.com/3046552/222838560-ea13246d-9391-4c6b-97ae-8f88f922f01c.png">

Here's the same screenshot with 600 points, pretty much identical.

<img width="815" alt="screenshot_2343" src="https://user-images.githubusercontent.com/3046552/222838593-05adf54a-a2d3-4d4e-951c-fbae890af80b.png">

Unlikely that the larger `lineWidth` would improve things either.

**Comment by pixelzoom** [comment]
Using `renderer: 'canvas'` addressed the problem.   I published [1.0.0-dev.19](https://phet-dev.colorado.edu/html/calculus-grapher/1.0.0-dev.19/phet/calculus-grapher_all_phet.html) with this fix.

Here's a screenshot from my iPad6:

![IMG_0048](https://user-images.githubusercontent.com/3046552/222918913-6fe0fd8c-1c8f-49ee-a73a-37db208da353.PNG)

Zooming in on that screenshot in macOS Preview:

<img width="509" alt="screenshot_2345" src="https://user-images.githubusercontent.com/3046552/222918938-60c2dd6a-b0c1-4e4e-a649-0b67a6853762.png">

Compared to zooming in on the screenshot that @liammulh provided:

<img width="437" alt="screenshot_2339" src="https://user-images.githubusercontent.com/3046552/222837021-f8b373c3-0e9d-4566-b529-691341d2db00.png">

And I confirmed that there's no noticeable difference (visual or performance) on other platforms.

Closing.
